data/templates/snmp/etc.snmpd.conf.j2

```diff
@@ -78,8 +78,8 @@ agentaddress unix:/run/snmpd.socket{{ ',' ~ options | join(',') if options is vy
 {%             endfor %}
 {%         endif %}
 {%         if comm_config.client is not vyos_defined and comm_config.network is not vyos_defined %}
-{{ comm_config.authorization }}community {{ comm }} -V RESTRICTED
-{{ comm_config.authorization }}community6 {{ comm }} -V RESTRICTED
+{{ comm_config.authorization }}community {{ comm }} 0.0.0.0/0 -V RESTRICTED
```

Comment:
This is an implicit default for the user and he will not know why. Please make this more transparent in using a `<defaultValue>` in the XML node, which will then also be added to the CLI help automatically. So the userbase will know that the network will default to 0.0.0.0/0 and ::/0.

Reply:
Is it a good idea to allow all addresses by default to get SNMP queries?
Or am I confusing something?

Reply:
IMO it's better to explicitly allow addresses.
This PR is just for backwards compatibility, so same configuration still works as it was working before latest changes in snmp.

But if we agree that it's better not to allow all access by default, we can close this PR, and prepare new updated and correct snmp documentation

Reply:
Moving this to `defaultValue` only adds "documentation" to the CLI and keeps the behavior "as is"

Reply:
Using multiple defaults is supported, see commit 011e6a2cdea8fca73f7c433de5e2d6bc22bd9c19.